tests/do_regtest.py

```diff
@@ -246,7 +247,7 @@ async def main() -> None:
 class Config:
     def __init__(self, args: argparse.Namespace):
         self.timeout = args.timeout
-        self.use_mpi = args.version.startswith("p")
+        self.use_mpi = args.mpi or args.version.startswith("p")
```

Comment:
Welp, probably this was the reason why all of the MPI tests were taking so long previously.